test/upgrade/simpleResources.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: tekton.dev/v1
+kind: Task
```

Comment:
can the task and pipeline be simplified a bit? I'm not sure we need to test `onError`, `stepTemplate`, etc. It's also a bit confusing to have unused workspaces and scripts with conditions; can these examples be made a bit more readable?

Reply:
updated the task names and usage of unnecessary fields.